.github/workflows/alpha.yml

```diff
@@ -75,7 +75,7 @@ jobs:
           args: release --snapshot --skip-publish
         env:
           COSIGN_PWD: ${{ secrets.COSIGN_PWD }}
-          GORELEASER_CURRENT_TAG: alpha
+          GORELEASER_CURRENT_TAG: v0.0.0-alpha
```

Comment:
What do you thing about generating the next major release using the alpha suffix? For example v2.6-alpha? I think that using v0.0.0 is confusing.

Reply:
In this workflow goreleaser it's being used just to build the binaries, so this tag its just to avoid the semver validation error.

After this step, another will run and publish a release with the alpha tag.

Reply:
Ohhh, I get it, make totally sense. I think this is fine.